tracing-subscriber/tests/vec_subscriber_filter_interests_cached.rs

```diff
@@ -4,14 +4,11 @@ use std::{
     sync::{Arc, Mutex},
 };
 use tracing::{Level, Subscriber};
-use tracing_mock::{
-    event,
-    layer::{self, MockLayer},
-};
+use tracing_mock::{layer::MockLayer, *};
 use tracing_subscriber::{filter, prelude::*};
 
 #[test]
-fn vec_layer_filter_interests_are_cached() {
+fn vec_subscriber_filter_interests_are_cached() {
```

Comment:
let's keep the v0.1.x naming here please (`vec_layer_filter_interests_are_cached`)